frontend/src/views/WheresMyGene/components/HeatMap/utils.ts

```diff
@@ -346,7 +346,8 @@ export function createYAxisOptions({
             const { name, depth = 0 } = deserializeCellTypeMetadata(
               value as CellTypeMetadata
             );
-
+            // TODO: We will have to add a ternary based on hover state too allow for room of the Marker Genes button
```

Comment:
Maybe I need to catch up on the context, but UX already showed that inline button for adding marker genes is hard to discover the feature, so I'm assuming we're not doing that anymore thus the comment is no longer relevant?

Reply:
Missed this file when cleaning up, ty!

Reply:
that's right!